content/2019-01-15_ctrnacty-sraz-vysvetlovani.md

```diff
@@ -0,0 +1,28 @@
+Title: Čtrnáctý sraz – opakování a vysvětlování
+Date: 2019-01-15 08:39:00
+Modified: 2019-01-15 08:39:00
+Author: Iveta Česalová
+
+
+
+Tento týden jsme se sešly v komorním počtu, navíc bez Petra. Ale cíl srazu byl jasný – abychom všechny rozuměly stávajícímu kódu.
+Začaly jsme souborem *util.py* a procházely metodu po metodě jednotlivých tříd. Vášnivá diskuze nastala u třídy HoleTile, která volá metodu robota – robot.die(). Původně bylo v kódu napsáno, že si robot volal metodu díry, což Petrovi přišlo zvláštní :-) Nyní je to tedy metoda políčka vyvolá metodu robota, což je logičtější, než aby si robot vyvolal metodu políčka :-D
```

Comment:
Tohle: "Nyní je to tedy metoda políčka vyvolá metodu robota" jako věta nedává moc smysl, napsala bych buď "Nyní je to tedy tak, že metoda políčka vyvolá metodu robota", nebo jen "Nyní tedy metoda políčka vyvolá metodu robota"